customer-satisfaction/README.md

```diff
@@ -28,9 +28,13 @@ zenml integration install mlflow -f
 
 ## :thumbsup: The Task
 
-We built an end-to-end pipeline for predicting the customer satisfaction score for the next order or purchase, which will help businesses make better decisions. We can't just train our model in our local system, and we need to serve the users, so we need it to be deployed in the cloud. We need machine learning pipelines for machine learning at scale, an end-to-end construct that orchestrates the flow of data into and output from a machine learning model (or set of multiple models). It includes raw data input, features, results, the machine learning model and model parameters, and prediction outputs. And all these capabilities are built on top of ZenML.
+We are tasked with building an end-to-end workflow for predicting the customer satisfaction score for the next order or purchase, which will help businesses make better decisions. 
```

Comment:
Don't you think it will be helpful for the user to explain why do we even need it? or what is this? assuming the user is beginner